ffl_predictor/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .team_summarizer import mean_var_scores
```

Comment:
Going to avoid doing that for now cause I want to avoid auto-exporting things on principle. Open to changing that later.